### DISCUSSION TITLE
LevelDB 源码分析「五、Sorted Table」 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by SF-Zhou** [SF-Zhou/sf-zhou.github.io#178] [Announcements]
https://sf-zhou.github.io/leveldb/leveldb_05_sorted_table.html

**Comment by IceHe** [comment]
很棒

**Comment by SF-Zhou** [comment]
> @IceHe
> 很棒

😳

**Comment by ikiwixx** [comment]
compact部分
...是否运行延迟、并且现在 L0 文件太多，如果是等 1 秒；....
是否 imm_ 还在 Compact 中，如果是就等待；...
此处我认为，L0文件太多属于等待
还有提个个人建议emm，第二个等待可以写详细些，就等待mutex_的锁，可以方便区分两个等待

**Comment by SF-Zhou** [comment]
> @ikiwixx
> compact部分
> ...是否运行延迟、并且现在 L0 文件太多，如果是等 1 秒；....
> 是否 imm_ 还在 Compact 中，如果是就等待；...
> 此处我认为，L0文件太多属于等待
> 还有提个个人建议emm，第二个等待可以写详细些，就等待mutex_的锁，可以方便区分两个等待

更新了表述。
